### DISCUSSION TITLE
Hi-C track: normalization and resolution

### DISCUSSION THREAD

**Discussion by ianakim** [GMOD/jbrowse-components#4610] [Q&A] [answered]
Hi, is it possible to change the applied normalization vector in the HicTrack config from KR to any other? Also, how can I set the default resolution at which to show the hiC data by default?

Thank you and best wishes,
Iana

**Comment by cmdcolin** [comment]
thanks for posting this. I think that neither of these is immediately possible. I will go ahead and re-open the issue you posted and it can be a feature request tracked from there :)

**Reply by ianakim** [reply]
thanks a lot, Colin!

**Reply by cmdcolin** [reply]
I made a proposal for this in  https://github.com/GMOD/jbrowse-components/pull/4628

example screenshot
![image](https://github.com/user-attachments/assets/0dc70ecc-f7b8-4fe6-b7e0-5f106a0300e7)

if you want to try out this branch and use jbrowse-web, you can use

```
jbrowse create --branch normalization_schemes_hic newinstance
```

this will get that PR beta branch. it has the ability to adjust the default resolution through what I refer to as a 'default multiplier' to the resolution. if instead you want to pick a precise bin size, that could maybe be done too, but the concept of resolution as just a sort of arbitrary multiplier has been in place for a little while now

so you could say, resolutionMultiplier of 10 (basically makes it think it is zoomed out by a factor of 10) on the adapter and it will show bigger squares by default. let me know if that seems ok

```json
{
      "type": "HicTrack",
      "trackId": "hic",
      "name": "Hic Track",
      "assemblyNames": ["hg19"],
      "adapter": {
        "type": "HicAdapter",
        "resolutionMultiplier": 10,
        "hicLocation": {
          "uri": "https://s3.amazonaws.com/igv.broadinstitute.org/data/hic/intra_nofrag_30.hic",
          "locationType": "UriLocation"
        }
      }
    }
```

![image](https://github.com/user-attachments/assets/781c0c00-6792-4c74-8cff-1753601732c4)


note a similar thing was done for bigwig also with resolution https://github.com/GMOD/jbrowse-components/pull/4619

**Reply by cmdcolin** [reply]
(note that I didn't add configuring normalization type yet but I could add that too potentially)

**Reply by cmdcolin** [reply, marked as answer]
released in v2.16.1, if you have a chance to try it out let me know
